### DISCUSSION TITLE
Is it possible to control a remote desktop through browser only without using any app like AnyDesk?

### DISCUSSION THREAD

**Discussion by umtcylmz** [ant-media/Ant-Media-Server#5541] [FAQs]
I am looking for a remote desktop solution which provides me the fastest way of connection without deploying any 3rd parties

**Comment by Mohit-3196** [comment]
[TunaDesk](https://antmedia.io/tunadesk/) is a promising solution for controlling a remote desktop through a web browser without relying on third-party apps. [TunaDesk](https://antmedia.io/tunadesk/) is a cross-platform, open-source remote desktop control tool designed to offer fast and secure access to remote computers while prioritizing user data privacy.

It offers unmatched benefits like:
- Complete Privacy
- Self-Hosted Solution
- Cross-Platform Compatibility
- Fast and Efficient
- Global Accessibility

Therefore, [TunaDesk](https://antmedia.io/tunadesk/) is a robust and secure solution for remote desktop control via web browsers. It empowers users to access their remote computers quickly and easily, all while ensuring the utmost privacy and control over their data. Whether you're working remotely, providing tech support, or simply need to access your computer from afar, [TunaDesk](https://antmedia.io/tunadesk/) offers a reliable and efficient solution without the need for third-party apps.

Thank you
